clarity/contracts/arkadiko-swap-v1-1.clar

```diff
@@ -24,6 +26,16 @@
 (define-constant no-fee-x-err (err u75))
 (define-constant no-fee-y-err (err u76))
 
+(define-data-var swap-shutdown-activated bool false)
+
+(define-public (toggle-swap-shutdown)
+  (begin
+    (asserts! (is-eq tx-sender (contract-call? .arkadiko-dao get-guardian-address)) (err ERR-NOT-AUTHORIZED))
```

Comment:
`tx-sender` should be replaced with `contract-caller` or you should add to this function 1 micro STX transfer force `post-conditions` usage.

https://app.sigle.io/friedger.id/HuOT9tNQC8fTXOsK28D7e